crates/system-services/src/deployer.rs

```diff
@@ -471,7 +471,13 @@ impl Deployer {
             // Stop old spell
             let result: eyre::Result<_> = try {
                 // Stop the spell to avoid re-subscription
-                self.call_service(&spell_id, "set_trigger_config", vec![json!(empty_config)])?;
+                let params = CallParams::local(
+                    spell_id.clone(),
+                    self.root_worker_id,
+                    Duration::from_millis(DEPLOYER_TTL),
+                );
+                self.spell_service_api
```

Comment:
maybe something like that will make it a single line?
```suggestion
                self.spells_api
```